homework-1/css/style.css

```diff
@@ -0,0 +1,1220 @@
+  padding: 24px 0 20px;
+}
+.fqas__link {
+  position: relative;
+  margin-top: 50px;
+}
+.fqas__link::before, .fqas__link::after {
+  content: "";
+  width: 10px;
+  height: 1.5px;
+  background-color: #37e63e;
+  position: absolute;
+  top: 20px;
+  transition: all 0.3s ease 0s;
+}
+.fqas__link::before {
+  transform: rotate(40deg);
+  right: -89px;
+}
+.fqas__link::after {
+  transform: rotate(-40deg);
+  right: -96px;
+}
+.fqas__link_active::before, .fqas__link_active::after {
+  background-color: #37e63e;
+}
+.fqas__link_active::before {
+  transform: rotate(-40deg);
+}
+.fqas__link_active::after {
+  transform: rotate(40deg);
+}
+.fqas-sub__item {
+  display: flex;
+  align-items: center;
+  margin-top: 17px;
+}
+.fqas-item__num {
+  font-family: "Actor-Regular";
+  font-size: 40px;
+  line-height: 48px;
+  text-align: center;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  color: #E5E5E5;
+}
+.fqas-item__title {
+  font-family: "Aclonica-Regular";
+  font-size: 16px;
+  line-height: 24px;
+  color: #191919;
+  width: 176px;
+  text-align: left;
+  margin-left: 30px;
+}
+.answer-title{
+  width: 241px;
+  font-size: 14px;
+  line-height: 24px;
+}
+.fqas-btn {
+  font-family: "Acme-Regular";
+  font-size: 12px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  color: #37E63E;
+  display: block;
+  margin: 0 auto;
+  padding-top: 72px;
+  transition: 1s;
+}
+.fqas-btn:hover {
+  color: #7f7f7f;
+}
+.footer {
+  background-color: #f2f2f2;
+  padding-top: 24px;
+}
+.footer__item-inner {
+  padding: 8px 0;
+  border-top: 2px solid #37E63E;
+}
+.footer__title {
+  font-size: 16px;
+  line-height: 36px;
+  color: #7F7F7F;
+}
+.menu-footer__list li + li {
+  padding-top: 10px;
+}
+.menu__link {
+  font-family: "Acme-Regular";
+  font-size: 12px;
+  line-height: 24px;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  color: #7F7F7F;
+}
+.phone {
+  margin-top: 4px;
+  margin-bottom: 10px;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7F7F7F;
+  transition: 1s;
+}
+.phone:hover {
+  color: #37e63e;
+}
+address {
+  font-size: 12px;
+  line-height: 20px;
+  color: #7F7F7F;
+  width: 170px;
+}
+.link a {
+  font-family: "Aclonica-Regular";
+  font-size: 12px;
+  line-height: 20px;
+  color: #37E63E;
+  transition: 1s;
+}
+.link a:hover {
+  color: #7f7f7f;
+  text-decoration: underline;
+}
+.social {
+  padding-bottom: 24px;
+}
+.social__inner {
+  border-top: 1.5px solid #b2b2b2;
+}
+.social-list {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  padding-top: 29px;
+  padding-bottom: 14px;
+}
+svg .social-hover {
+  transition: 200ms;
+  transition-timing-function: ease-in-out;
+  -webkit-transition: 200ms;
+  -webkit-transition-timing-function: ease-in-out;
+}
+svg:hover .social-hover {
+  fill: #37E63E;
+}
+.mobile__img {
+  display: block;
+  margin: 0 auto;
+  height: 38px;
+}
+.dekstop__img {
+  display: none;
+}
+@media (min-width: 640px) {
```

Comment:
It will be enough to use breakpoints - 768,  1200, 1440

Reply:
fixed a bug